fixed issues 17 and 18

LyzandeR opened the pull request:
Added the code that takes care of both issues #17  and #18 .

Checks seem to be passing as well.

LyzandeR commented:
@clemens-zauchner Do you want to check if the function `tableHTML_to_image` works fine with you as well and also check the new `round` and `replace_NA` arguments of `tableHTML`?? 

I have run some tests myself and added to tests/ as well.

Then I can merge to master.

LyzandeR commented:
Also added a correction for #23 . It seems to be working now even for same column names.

clemens-zauchner commented on R/tableHTML2image.R:
```diff
@@ -0,0 +1,94 @@
+ #temp file to save the image
+ image <- tempfile(fileext = paste0('.', type))
+
+ #writing tableHTML HTML into the temp file
```
what is the point of this if statement? it has the same code in both `if` and `else`

LyzandeR replied:
You are absolutely right on this. You added the css for the jpeg tables to avoid the black background and then I noticed that when printing a png on a file, there was no background, so I added the same code. So, no distinction is needed :P. I will remove the if-else and repush

LyzandeR replied:
Left the if-else statement there adding white background for jpeg and left png without one.

clemens-zauchner commented on R/tableHTML2image.R:
```diff
@@ -0,0 +1,94 @@
+  return(grid::grid.raster(img))
+ } else {
+  writefunc(img, file)
+  return(NULL)
```
what about `invisible(NULL)` instead of `return(NULL)`?

LyzandeR replied:
definitely. Will add and push.

LyzandeR replied:
Done

LyzandeR commented:
Added examples to the examples vignette, updated the documentation, fixed the code you mentioned in the review. I think we should be ready to merge now. What do you think?
